recipes/libcurl/all/conanfile.py

```diff
@@ -226,7 +226,7 @@ def layout(self):
     def source(self):
         get(self, **self.conan_data["sources"][self.version],
                   destination=self.source_folder, strip_root=True)
-        download(self, "https://curl.haxx.se/ca/cacert.pem", "cacert.pem", verify=True, sha256="2cff03f9efdaf52626bd1b451d700605dc1ea000c5da56bd0fc59f8f43071040")
+        download(self, "https://curl.se/ca/cacert-2023-01-10.pem", "cacert.pem", verify=True, sha256="fb1ecd641d0a02c01bc9036d513cb658bbda62a75e246bedbc01764560a639f0")
```

Comment:
Wouldn't it be better to download the last file + the corresponding checksum ( https://curl.se/ca/cacert.pem.sha256 )?

Reply:
One purpose of the checksum is to detect malicious tampering or modification if, at some time in the future, the server you're downloading the file from is compromised. It will no longer achieve this if you're just trusting the same server you're downloading the file from to give you the correct checksum.

Reply:
Ok, thanks for explanation. It seems like it's worth doing this manually from time to time.